README.md

```diff
@@ -0,0 +1,58 @@
+# Clear Containers Tests Repository
+
+This repository contains the test suite for the Clear Containers 3.0 project,
+including functional and integration tests with Docker\*, Kubernetes\* and OpenShift\*.
+
+An environment with Clear Containers properly setup is needed in order to execute 
+these tests. If you require to setup Clear Containers, please refer to the:
```

Comment:
"If you require" is passive, I suggest we re-word:
For instructions on how to setup Clear Containers, please refer to the [Installation Guides](https:...).